extensions/roc-package-web-app-react/src/app/server/reactRenderer.js

```diff
@@ -81,6 +106,7 @@ export function initRenderPage({ script, css }, distMode, devMode, Header) {
             serializedRocConfig: serialize(rocConfigClient),
             status,
             styleName,
+            templateContext: build.templateContext,
```

Comment:
It seems like `templateContext` should be removed completely, since `get-template-values` hook is more flexible and no version with `templateContext` support was released.